tests/test_wave_03.py

```diff
@@ -54,13 +54,10 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert FANTASY_1 not in friends_unique_movies
+    assert INTRIGUE_3 in friends_unique_movies
```

Comment:
This is good! Also you could test for the other two movies that are supposed to be in `friends_unique_movies`